packages/cms-cli/commands/hubdb.js

```diff
@@ -44,7 +44,9 @@ function configureHubDbCreateCommand(program) {
       setLogLevel(command);
       logDebugInfo(command);
       const { config: configPath } = command;
-      loadConfig(configPath);
+      loadConfig(configPath, {
+        allowEnvironmentVariableConfig: true,
```

Comment:
Why do we need this flag?

Reply:
If we try running `hs auth` with environmental variable data present, it never writes to file.

Reply:
Ah, I understand the complexity now. I would think that we would treat running commands that don't support environment variables as an error and exit early since it would be a mistake to run `hs init` or `hs auth` with environment variables.

Reply:
I think with the ability to use `HUBSPOT_PORTAL_ID` similarly to `defaultPortal`, the possibility of wanting to run `hs auth` with `HUBSPOT_PORTAL_ID` set is much higher. The ability to handle this case gracefully is preferable to asking the user to `unset HUBSPOT_PORTAL_ID`, re-run `hs auth`, and `export HUBSPOT_PORTAL_ID` again.

With the new code, this case is handled so the user can run `hs auth` with `HUBSPOT_PORTAL_ID` set.